components/writers/src/main/java/org/datacleaner/extension/output/CreateCsvFileAnalyzer.java

```diff
@@ -129,40 +147,37 @@ public void configureForTransformedData(AnalysisJobBuilder ajb, TransformerDescr
     }
 
     @Override
-    public String getSuggestedLabel() {
-        if (file == null) {
-            return null;
-        }
-        return file.getName();
-    }
-
-    @Override
     public OutputWriter createOutputWriter() {
-        List<String> headers = new ArrayList<String>() ;
+        
+        if (file.exists() && overwriteFile==false){
```

Comment:
Yeah, but validate it is not called in Save us utilities. 
![image](https://cloud.githubusercontent.com/assets/6339923/5959150/efa05606-a7cb-11e4-85d2-cc61f4374f6c.png)